src/sparseml/pytorch/image_classification/utils/trainer.py

```diff
@@ -170,6 +172,10 @@ def run_one_epoch(
         train_mode = mode == "train"
         validation_mode = not train_mode
 
+        if torch.__version__ < "1.9" and self.manager.qat_active(epoch=self.epoch):
```

Comment:
> Noting that torch > 1.9. does not crash if mixed precision is not switched off before QAT start
Does torch == 1.9.x crash? Because it would pass above this check

Reply:
It wouldn't pass the above check I confirmed:
```python
In [1]: "1.9.1" < "1.9"
Out[1]: False
```

Anything >1.9 dos not need mixed precision to be turned off so it's good